src/View/TemplatePaths.php

```diff
@@ -258,14 +258,14 @@ public function resolveTemplateFileForControllerAndActionAndFormat($controller,
 	}
 
 	/**
-	 * @param string $controllerName
+	 * @param string|NULL $controllerName
 	 * @param string $format
 	 * @return array
 	 */
 	public function resolveAvailableTemplateFiles($controllerName, $format = self::DEFAULT_FORMAT) {
```

Comment:
I hesitate to introduce that change just now, as it would be a breaking change for PHP7 on platforms providing a custom TemplatePaths implementation - but you are correct and we should do this at some point (and then bump to version 2.0). But let's accumulate some more breaking changes before we do that ;)